_pytest/fixtures.py

```diff
@@ -553,7 +553,7 @@ def _getscopeitem(self, scope):
         if node is None and scope == "class":
             # fallback to function item itself
             node = self._pyfuncitem
-        assert node
+        assert node, 'Could not obtain a node for scope "{}" for function {!r}'.format(self._pyfuncitem)
```

Comment:
Shouldn't there be two args to `format()`? (`scope` is missing)

Reply:
Definitely, thanks for catching it! 😉 

That's your morning coffee not kicking in yet. 😅